addons/web/static/src/views/view_service.js

```diff
@@ -74,7 +74,8 @@ export const viewService = {
                 loadViewsOptions.mobile = true;
             }
             const { context, resModel, views } = params;
-            const key = JSON.stringify([resModel, views, context, loadViewsOptions]);
+            let filteredContext = Object.fromEntries(Object.entries(context).filter((k,v) => !String(k).startsWith("default_")));
```

Comment:
can the filtering be done upstream instead? I assume it needs to be done by a owl_view, that can filter the context themselves